README.md

```diff
@@ -1,8 +1,8 @@
 ***
-Java code introduction for beginners workshop.
+Java code instrumentation for beginners workshop.

```

Comment:
This should be in Title Case: "Java Code Instrumentation for Beginners Workshop"